JDE insists on seeing CEDET's source tree layout, with various
sub-directories (`common', `eieio', etc.).  However, the installed CEDET
is flat, with everything under ${cedet}/share/emacs/site-lisp.

--- jde/config/build.el	(revision 90)
+++ jde/config/build.el	(working copy)
@@ -50,10 +50,5 @@ PATHS are sub directories under CEDET-DI
 (jde-make-autoloads-and-compile (expand-file-name "@{build.lisp.dir}")
 				"@{src.lisp.dir}"
 				"@{cedet.dir}"
-				'("common"
-				  "eieio"
-				  "semantic"
-				  "semantic/bovine"
-				  "speedbar"
-				  )
+				'(".")
 				"@{build.lisp.autoload.libname}")
